randovania/games/prime1/layout/prime_configuration.py

```diff
@@ -21,6 +21,20 @@ class LayoutCutsceneMode(BitPackEnum, Enum):
     MINOR = "minor"
     MAJOR = "major"
 
+class EnemyAttributeRandomizer(BitPackDataclass):
+    optional_int: int | None = dataclasses.field(metadata={"min": 0, "max": 15})
```

Comment:
I don't see a purpose for this?

Reply:
Whoops, I copied the values from test for reference but I forgot to delete it.